cdci_data_analysis/flask_app/dispatcher_query.py

```diff
@@ -809,10 +814,13 @@ def find_job_id_parameters(self, job_id):
         else:
             return json.load(open(scratch_dir_parameters[0]))
 
-
     # TODO make sure that the list of parameters to ignore in the frontend is synchronized
     def generate_products_url_from_par_dict(self, products_url, par_dict) -> str:
         par_dict = par_dict.copy()
+        if 'scw_list' in par_dict and self.use_scws is not None:
```

Comment:
actually, if `self.scw_list` is already set by the function above, maybe it can be used here?

Reply:
`self.scw_list` does not actually exist, or am I wrong?